morpheus/messages/message_meta.py

```diff
@@ -80,6 +82,13 @@ def __init__(self, df: pd.DataFrame) -> None:
         self._mutex = threading.RLock()
         self._df = df
 
+    @classmethod
+    def make_from_file(cls, filename: str):
```

Comment:
Just not sure I love this method. It doesnt allow you to create any other classes besides `MessageMeta`. Also it doesnt forward on arguments to the constructor of the class type. And finally, it would prevent using the argument `filename` in derived constructors if you worked around the other issues.

Reply:
Yeah, that's fair. The alternative would be to expose the C++ deserializers to Python, such that `read_file_to_df` will call the C++ impl when `df_type="cudf"`.

The down side is it would be an extra trip back to python to return the dataframe, and then back in to C++ to construct the `MessageMeta`.

Personally I do a lot of testing/debugging from `ipython` and I typically need to do:
```python
from morpheus._lib.common import FileTypes
from morpheus.io.deserializers import read_file_to_df
from morpheus.messages import MessageMeta

m = MessageMeta(read_file_to_df('tests/tests_data/filter_probs.csv', file_type=FileTypes.Auto, df_type='cudf'))
```

With this `classmethod` that trims down to:
```python
from morpheus.messages import MessageMeta
m = MessageMeta.make_from_file('tests/tests_data/filter_probs.csv')
```